code/game/objects/items/weapons/implants/implant.dm

```diff
@@ -351,33 +351,21 @@ the implant may become unstable and either pre-maturely inject the subject or si
 		malfunction--
 
 /obj/item/implant/loyalty
```

Comment:
This isn't really a `/loyalty` implant anymore, is it?

Reply:
Should be done.